util/package.go

```diff
@@ -182,22 +182,19 @@ func (p *Package) LoadAssets(packagePath string) (err error) {
 		return err
 	}
 
-	assets, err := filepath.Glob("*")
-	if err != nil {
-		return err
-	}
-
-	a, err := filepath.Glob("*/*")
-	if err != nil {
-		return err
-	}
-	assets = append(assets, a...)
-
-	a, err = filepath.Glob("*/*/*")
-	if err != nil {
-		return err
+	var assets []string
+	var pattern = "*"
+	// Iterates 6 levels deep through the tree to find assets
```

Comment:
Will there be a time when 6 levels aren't deep enough? Is there a way to solve this recursively once and for all?

Reply:
I'm sure it will happen. There is a library that does it but I was hesitant to include it here: https://github.com/bmatcuk/doublestar I'm just thinking of an alternative recursive implementation now that I mention it which "should" always work. Will update.